tests/test_parser.py

```diff
@@ -0,0 +1,41 @@
+# coding=utf-8
+import unittest
+from collections import OrderedDict
+
+from text2chem.core.formula_parser import __parse_parentheses as parse_parentheses
+from text2chem.regex_parser import RegExParser
+
+
+class TestParser(unittest.TestCase):
+
+    def test_regex_parser_make_fraction_convertion(self):

```

Comment:
Please split each formula into a single method, so that the failing formulas / methods are quickly identified and there are no hidden issues.